keps/sig-api-machinery/4192-svm-in-tree/README.md

```diff
@@ -125,59 +125,59 @@ As an end user using encryption at rest, whenever the key change is detected we
     // StorageVersionMigration represents a migration of stored data to the latest
     // storage version.
     type StorageVersionMigration struct {
-        metav1.TypeMeta `json:",inline"`
-        // +optional
-        metav1.ObjectMeta `json:"metadata,omitempty"`
-        // Specification of the migration.
-        // +optional
-        Spec StorageVersionMigrationSpec `json:"spec,omitempty"`
-        // Status of the migration.
-        // +optional
-        Status StorageVersionMigrationStatus `json:"status,omitempty"`
+      metav1.TypeMeta `json:",inline"`
```

Comment:
Can't comment on unchanged lines.

We were chatting about streaming lists in this context during last kubecon with @deads2k and @enj .
Please correct me if I'm wrong, but the conclusion that we got to was that given:
- we're moving SVM to kcm component
- kcm already has an internal cache for GC purposes

We're not going to use streaming lists, but rather stick to taking advantage of the KCM cache to get the list of objects that needs to be updated.

Reply:
@wojtek-t That is correct. I have updated the KEP the reflect this. Slack thread - https://kubernetes.slack.com/archives/C0EG7JC6T/p1706033020053539